tueplots/fonts.py

```diff
@@ -3,147 +3,181 @@
 
 
 def jmlr2001_tex(*, family="serif"):
     """Fonts for JMLR. LaTeX version."""
-    return _tex_computer_modern(family=family)
+    return _computer_modern_tex(family=family)
 
 
 def tmlr2023_tex(*, family="serif"):
     """Fonts for TMLR. LaTeX version."""
-    return _tex_computer_modern(family=family)
+    return _computer_modern_tex(family=family)
 
 
 def aistats2022_tex(*, family="serif"):
     """Fonts for AISTATS 2022. LaTeX version."""
-    return _tex_computer_modern(family=family)
+    return _computer_modern_tex(family=family)
 
 
 def aistats2023_tex(*, family="serif"):
     """Fonts for AISTATS 2023. LaTeX version."""
-    return _tex_times(family=family)
+    return _times_tex_via_pkg_times(family=family)
 
 
 def uai2023_tex(*, family="serif"):
     """Fonts for UAI 2023. LaTeX version."""
-    return _tex_times(family=family)
+    return _times_tex_via_pkg_times(family=family)
 
 
 def aaai2024_tex(*, family="serif"):
     """Fonts for AAAI 2024. LaTeX version."""
-    return _tex_times(family=family)
+    return _times_tex_via_pkg_times(family=family)
 
 
-def _tex_times(*, family):
-    preamble = r"\usepackage{times} "
+def icml2022(*, family="serif"):
+    """Fonts for ICML 2022."""
+    return _times(family=family)
+
+
+def beamer_moml():
+    """Fonts that are compatible with the beamer template of the method-of-machine-learning group in Tübingen."""
+    return {
+        "text.usetex": False,
+        "mathtext.fontset": "custom",
+        "mathtext.it": "sans:italic",
+        "font.sans-serif": ["Roboto Condensed"],
+        "font.family": "sans-serif",
+        "font.weight": "light",
+        "axes.labelweight": "light",
+        "axes.titleweight": "light",
+    }
+
+
+def beamer_moml_dark_bg():
+    """Fonts for :func:`beamer_moml` with dark background."""
+    return {
+        "text.usetex": False,
+        "font.serif": ["Roboto Condensed"],
+        "font.family": "serif",
+        "font.weight": "light",
+        "axes.labelweight": "light",
+        "axes.titleweight": "light",
+    }
+
+
+# Helper functions below
+
+
+def _times(*, family="serif"):
+    """Times font.
+
+    Used, e.g., for ICML 2022.
+    """
+
+    return {
+        "text.usetex": False,
+        "font.serif": ["Times"],
+        "mathtext.fontset": "stix",  # free ptmx replacement, for ICML and NeurIPS
+        "mathtext.rm": "Times",
+        "mathtext.it": "Times:italic",
+        "mathtext.bf": "Times:bold",
+        "font.family": family,
+    }
+
+
+# def _times_new_roman(*, family="serif"):
+#     """Default fonts for Neurips."""
+#     return {
+#         "text.usetex": False,
+#         "font.serif": ["Times New Roman"],
+#         "mathtext.fontset": "stix",  # free ptmx replacement, for ICML and NeurIPS
+#         "mathtext.rm": "Times New Roman",
+#         "mathtext.it": "Times New Roman:italic",
+#         "mathtext.bf": "Times New Roman:bold",
+#         "font.family": family,
+#     }
+
```

Comment:
Let's delete this; otherwise, it looks great!

Reply:
Okay! Just pushed the changes :)